hack/build-local-images.py

```diff
@@ -170,11 +170,16 @@ def add_to_context(context_dir, source, destination, container_destination):
 		container_destination)
    	)
     absolute_destination = abspath(join(context_dir, destination))
-    if not exists(absolute_destination):
-        if isdir(source):
-            copytree(source, absolute_destination)
-        else:
-            copy(source, absolute_destination)
+    if isdir(source):
+        for filename in listdir(source):
+            full_filename = join(source, filename)
+            full_destination = join(absolute_destination, filename)
+            if (isdir(full_filename)):
+                copytree(full_filename, full_destination)
```

Comment:
Err -- this should be recursive. But why not make sure `full_destination` doesn't exist? Where exactly it is doesn't matter, it just needs to be a path we can put into the `Dockerfile` in the `ADD`.